map.go

```diff
@@ -568,8 +587,10 @@ func (m *Map[K, V]) uncheckedPut(h uintptr, key K, value V) {
 			slot := m.slots.At(i)
 			slot.key = key
 			slot.value = value
+			if *m.ctrls.At(i) == ctrlEmpty {
+				m.growthLeft--
```

Comment:
This was a tiny bug the invariant checking caught. If the slot was previously deleted we should not have been decrementing `growthLeft`. The impact would be rehashing/resizing slightly more often, though I couldn't measure any effect on the benchmarks.